### PR TITLE
finally working updates

### PR DESCRIPTION
**Pull request by mkveerapen**

### CODE REVIEW

File: notebook/images/hail/Dockerfile [hunk 1]
```diff
@@ -8,8 +8,10 @@ RUN apt-get update && apt-get install -y \
 USER jovyan
 
 RUN pip install --no-cache-dir \
-  'jupyter-spark<0.5' \
-  hail==0.2.8 \
+  jupyter \
+  jupyter-spark \
+  "tornado<6" \
+  hail==0.2.49 \
```
Comment (johnc1231) [inline notebook/images/hail/Dockerfile:14]: 0.2.54, I want the newest version here.

File: site/www/references.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ Otherwise, we welcome you to add additional examples by [editing this page direc
 
 ## 2020
 
+* Krissaane, I, et al. "Scalability and cost-effectiveness analysis of whole genome-wide association studies on Google Cloud Platform and Amazon Web Services", *Journal of the American Medical Informatics Association* (2020) ocaa068 <https://doi.org/10.1093/jamia/ocaa068> <https://academic.oup.com/jamia/article/doi/10.1093/jamia/ocaa068/5876972>
```
Comment (johnc1231) [inline site/www/references.md:28]: This really should be in a separate PR. Each PR should do its own thing. This is a mix of workshop stuff and references. It's fine this time, but in the future don't do this.

Comment (mkveerapen) [reply site/www/references.md:28]: Will do! So next time, workshop is one PR and a second PR for reference edits?

File: notebook/images/hail/Dockerfile (hunk 1 above)
Comment (mkveerapen) [reply notebook/images/hail/Dockerfile]: done!

**Pull request merged**